src/java/org/apache/cassandra/tools/NodeProbe.java

```diff
@@ -2034,14 +2036,17 @@ public int getDefaultKeyspaceReplicationFactor()
         return ssProxy.getDefaultKeyspaceReplicationFactor();
     }
 
-    public void setMinimumKeyspaceReplicationFactor(int value)
+    public void setMinimumKeyspaceReplicationFactor(int warn, int fail)
```

Comment:
Currently all guardrails can be dynamically changed through direct JMX methods, without offering nodetool setters nor setters. Instead, there are plans to make them accessible through virtual tables, although sadly that will be after 4.1. Even if we wanted to add nodetool setters/getters, we could consider adding some `nodetool setguardrail/getguardrail` common command. Thus, I wonder if we want to keep the `nodetool getdefaultrf/setdefaultrf` commands added by CASSANDRA-14557 or just remove them. @dcapwell wdyt?

Reply:
> Also, added nodetool commands to override default rf and minimum rf. This is still cumbersome to do it for each node, but yes it serves the purpose of avoiding node restart. We may ideally want to put an endpoint in the [sidecar](https://github.com/apache/cassandra-sidecar), that could potentially change these configurations across the cluster.

from 14557, sounds like its there for simplicity and not sure sidecar ever got the functionality.

I am +0 to removing with a slight preference to keep.

@sumanth-pasupuleti  as the author what is your stance here?  The TL;DR is that your min_rf feature is now min_rf_warn_threshold and min_rf_fail_threshold, which is now awkward in nodetool (doable, just awkward)